certbot/auth_handler.py

```diff
@@ -100,10 +100,16 @@ def _choose_challenges(self, domains):
         """Retrieve necessary challenges to satisfy server."""
         logger.info("Performing the following challenges:")
         for dom in domains:
+            dom_challenges = self.authzr[dom].body.challenges
+            if self.acme.acme_version == 1:
+                combinations = self.authzr[dom].body.combinations
+            else:
+                combinations = tuple((i,) for i in range(len(dom_challenges)))
```

Comment:
the documentation below says combinations is a tuple, but for some reason I thought it was a list. obviously it should work either way, but were you basing this off the docs or manually checking?

Reply:
Manually checking against the V1 endpoint. It was a tuple of tuples.